pkgs/sops-install-secrets/nixos-test.nix

```diff
@@ -203,110 +203,4 @@
    inherit pkgs;
    inherit (pkgs) system;
  };
-
-} // pkgs.lib.optionalAttrs (pkgs.lib.versionAtLeast (pkgs.lib.versions.majorMinor pkgs.lib.version) "21.11") {
```

Comment:
To avoid merge conflicts, we can keep the test code here, with a comment that we currently not officially support this feature.